src/Language/PureScript/Ide/Error.hs

```diff
@@ -51,9 +51,9 @@ encodeRebuildErrors = toJSON . map encodeRebuildError . P.runMultipleErrors
       (P.ErrorMessage _
        ((P.HoleInferredType name _ _
          (Just P.TSAfter{tsAfterIdentifiers=idents, tsAfterRecordFields=fields})))) ->
-        insertTSCompletions name idents (fromMaybe [] fields) (toJSON (toJSONError False P.Error err))
+        insertTSCompletions name idents (fromMaybe [] fields) (toJSON (toJSONError False P.Error [] err))
       _ ->
-        (toJSON . toJSONError False P.Error) err
+        (toJSON . toJSONError False P.Error []) err
```

Comment:
This means that any error messages that are generated through this code path don't include the new highlighted source file excerpts. The PureScript IDE integration is something I don't have experience with as a user or maintainer, so this could be good or bad depending on where in the UI this stuff goes.

Reply:
This is the only thing stopping me from approving this PR, and mainly because I'm unfamiliar with this code and the resulting implications.

Reply:
Oh, just saw the other commit. Nevermind!